### PR TITLE
CLDR-13720 shorten narrow display names for digital units

### PR DESCRIPTION
**Pull request by macchiati**
##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-13720
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

**Comment by macchiati** [comment]
Note: all of them were shortened except for "bit".

File: common/main/en.xml [hunk 1]
```diff
@@ -8015,52 +8015,52 @@ annotations.
...
 			</unit>
 			<unit type="digital-kilobit">
-				<displayName>kbit</displayName>
+				<displayName>kb</displayName>
 				<unitPattern count="one">{0}kb</unitPattern>
 				<unitPattern count="other">{0}kb</unitPattern>
 			</unit>
 			<unit type="digital-byte">
-				<displayName>byte</displayName>
+				<displayName>B</displayName>
```
Comment (sffc) [inline common/main/en.xml:8063]: Why did you change `byte` to `B` but not `bit` to `b`?

Comment (robertgmelo) [reply common/main/en.xml:8063]: Maybe because narrow values on bit type are `bit`, so I assume Mark wanted to keep the logic same as other units. In any case, it's a good question. Maybe if we change _displayName_ from `bit` to `b`, we would also need to change the values, but Mark can confirm.
https://github.com/unicode-org/cldr/blob/master/common/main/en.xml#L8069-L8070

Comment (macchiati) [reply common/main/en.xml:8063]: The question is whether "b" would be recognizable in isolation: "3 b" or "3b". The kb or Mb is distinctive enough, but I'm a bit concerned about the singleton 'b'. But maybe I shouldn't be?

Comment (pedberg-icu) [reply common/main/en.xml:8063]: I think you should be. I do not think b is distinctive. I am not even sure that B by itself is distinctive.

**Comment by macchiati** [comment]
We need to decide this before Wednesday if we want to get into 38.

**Comment by macchiati** [comment]
Ok, I'll move the ticket out of design and merge
